tests/console_test.py

```diff
@@ -14,10 +14,15 @@
 
 from click.testing import CliRunner
 from utilities_common.db import Db
-from consutil.lib import *
+from consutil.lib import ConsolePortProvider, ConsolePortInfo, ConsoleSession, SysInfoProvider, DbUtils, \
+    InvalidConfigurationError, LineBusyError, LineNotFoundError, ConnectionFailedError
```

Comment:
Some Errors seem not to be used

Reply:
All these are used. If not, it would be caught by flake8 check.